src/components/side_panel/more_formats/more_formats.xml

```diff
@@ -0,0 +1,36 @@
+<templates>
+  <t t-name="o-spreadsheet-MoreFormatsPanel">
+    <div class="o-more-formats-panel">
+      <div class="fw-bold m-3">Dates</div>
+      <div class="d-flex flex-column">
+        <button
+          t-foreach="dateFormats"
+          t-as="action"
+          t-key="action.description(env)"
+          t-on-click="() => action.execute(env)"
+          class="w-100 btn btn-outline-light d-flex align-items-center"
+          t-att-title="action.name(env)">
+          <span class="float-start ms-3 check-icon">
```

Comment:
is float-start ever useful ?
I think i'd go with less/no margin start, I think it'd be prettier